library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -5,12 +5,11 @@ task CalculateGeneMetrics {
   command {
     set -e
 
-    CalculateGeneMetrics -i "${bam_input}" -o gene-metrics.csv
-    gzip gene-metrics.csv
+    CalculateGeneMetrics -i "${bam_input}" -o gene-metrics.csv.gz
```

Comment:
Correct, I adjusted the CalculateGeneMetrics function in the `sctools` PR to output gzipped files instead, since this is easier on disk space and avoids the necessity of post-hoc zipping the files.

Reply:
👍